backend/src/main/java/com/github/liliancharlotte/model/User.java

```diff
@@ -1,9 +1,14 @@
 package com.github.liliancharlotte.model;
 
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
 import java.util.List;
 
+@Document("users")
 public record User(
+        @Id
         String id,
         String name,
         List<JobPosting> jobPostings) {
-}
+}
```

Comment:
Eine uralte Konvention für Textdateien besagt, dass in Textdateien jede Zeile mit einem Zeilenumbruch enden soll (auch die letzte Zeile). Viele Tools, wie z.B. GitHub oder Linux-Kommandozeilentools rechnen daher mit einem Zeilenumbruch am Ende der Datei. Das Fehlen in dieser Datei wird von GitHub mit einem 🚫 gekennzeichnet.

Kein großes Ding - nur ein klitzekleiner Schönheitsmakel.
```suggestion
}

```